internal/dlasqtest/commands.txt

```diff
@@ -0,0 +1,9 @@
+Advice: nDiv in dlasq2 can be an effective counter for isolating a bug. Using os.Exit(1) and "STOP" when nDiv hits a certain number.
```

Comment:
Make this something that can be executed, or at least sourced - prefix this line with `#`.